src/utils/colors/index.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Utility function to apply transparency to a color (variable) using `color-mix`
+ * @param {string} color - The color to make transparent. Can be a theme variabale, a hex code or "currentColor"
+ * @param {number} opacity - The desired opacity, a number between 0 and 1
+ */
```

Comment:
I think this should live inside the commit message instead of a comment.

Reply:
The plus side of this for me was that the tooltip of the function will explain what is needed, which I thought would be helpful for the `opacity` specifically (there are a number of correct ways to pass that through). Any alternative method for that?

Reply:
I do like it tbh, but since it's not a practice in my codebase it seems a bit inconsistent to me 🤷 but yeah not sure what the others think.